src/AppBundle/Entity/Adherent.php

```diff
@@ -657,21 +657,15 @@ public function isSupervisorOf(Committee $committee): bool
 
     public function setLegislativeCandidate(?string $area): void
     {
-        if (!$area) {
-            $this->legislative = null;
-
-            return;
+        if (!$this->legislative) {
```

Comment:
Yes but I don't want to create empty LegislativeCandidate on each adherent edit.